findspam.py

```diff
@@ -407,6 +407,19 @@ def strip_urls_and_tags(string):
     return regex.sub("</?.+?>|\w+?://", "", regex.sub(URL_REGEX, "", string))
 
 
+# noinspection PyUnusedLocal,PyMissingTypeHints
+def mostly_dots(s, site, *args):
+    body = strip_urls_and_tags(s)
+    body_length = len(body)
+
+    dot_count = len(regex.findall("\.", body))
+
+    if body_length and dot_count / float(body_length) >= 0.4:
```

Comment:
We can always *import* `__future__` but we'd have to redo a large portion of our math.  I only recently starting using the division aspect of `__future__` in one of my other scripts, but so far I'm a fan of it.  I'm not a fan of some other aspects of Python 3 though.  Yet.